sql/step_94_missing_user_related_fk.sql

```diff
@@ -0,0 +1,50 @@
+ALTER TABLE `oh_admission` 
```

Comment:
in scripts we use to CAPITALIZE all name tables and to remove single quotes

Reply:
Fixed with commit [a4d091e](https://github.com/informatici/openhospital-core/pull/998/commits/a4d091e7703a11350f77a67450fcab1b5e17ab82)